### ISSUE TITLE
broken paths in map tempate

### ISSUE THREAD

**Issue by andybell** [ucd-cws/arcproject-wq-processing#98]
ArcMap output from Generate Map for Export has the following broken paths:

- [ ] Arc_DeltaWaterways_0402
- [ ] sj_maj_channels_0402

These should be added to the geo folder

**Comment by nickrsan** [comment]
Yeah, I noticed this earlier today when I was debugging the map exports. I turned off relative paths in the template, and it seemed to fix the issue. I'm about to commit what I think will fix a few mapping issues. If you'd merge across and test it again, I'd appreciate it.

**Comment by nickrsan** [comment]
OK, just pushed a changeset that should make map exports work normally on ArcMap. Give that a try and let me know if you see issues.

**Comment by nickrsan** [comment]
dangit - I screwed that up. Made that change on the testing machine, but not the development machine. One moment

**Comment by nickrsan** [comment]
OK, *now* I've pushed a changeset where mapping should work properly.

**Comment by nickrsan** [comment]
Confirmed that these were still broken when I did my video demo. I'll investigate. This is the part of ArcGIS that I hate :/
